packages/synapse-react-client/src/synapse-queries/subscription/useSubscription.ts

```diff
@@ -87,24 +165,38 @@ export function usePostSubscription(
 }
 
 export function useDeleteSubscription(
-  options?: UseMutationOptions<void, SynapseClientError, string>,
+  options?: UseMutationOptions<void, SynapseClientError, Subscription>,
 ) {
   const queryClient = useQueryClient()
   const { accessToken, keyFactory } = useSynapseContext()
 
-  return useMutation<void, SynapseClientError, string>(
-    (subscriptionId: string) =>
-      SynapseClient.deleteSubscription(accessToken, subscriptionId),
+  return useMutation<void, SynapseClientError, Subscription>(
+    (subscription: Subscription) =>
+      SynapseClient.deleteSubscription(
+        accessToken,
+        subscription.subscriptionId,
+      ),
     {
       ...options,
       onSuccess: async (updatedSubscription, variables, ctx) => {
-        await queryClient.invalidateQueries(
-          keyFactory.getAllSubscriptionsQueryKey(),
-        )
+        await Promise.all([
+          queryClient.invalidateQueries(
+            keyFactory.getAllSubscriptionsQueryKey(),
+          ),
+          queryClient.invalidateQueries(
+            keyFactory.getSubscriptionQueryKey(
+              variables.objectId,
+              variables.objectType,
+            ),
+          ),
+          queryClient.invalidateQueries(
+            keyFactory.getSubscribersQueryKey(
+              variables.objectId,
+              variables.objectType,
+            ),
+          ),
+        ])
```

Comment:
Ditto above, no longer invalidate queries for individual subscriptions besides this one.